internal/services/netapp/netapp_volume_resource_test.go

```diff
@@ -16,7 +16,39 @@ import (
 
 type NetAppVolumeResource struct{}
 
-func TestAccNetAppVolume_basic(t *testing.T) {
+func TestAccNetAppVolume(t *testing.T) {
+	testCases := map[string]map[string]func(t *testing.T){
+		"Resource": {
+			"basic":                              testAccNetAppVolume_basic,
+			"nfsv41":                             testAccNetAppVolume_nfsv41,
+			"standardNetworkFeature":             testAccNetAppVolume_standardNetworkFeature,
+			"snapshotPolicy":                     testAccNetAppVolume_snapshotPolicy,
+			"crossRegionReplication":             testAccNetAppVolume_crossRegionReplication,
+			"nfsv3FromSnapshot":                  testAccNetAppVolume_nfsv3FromSnapshot,
+			"nfsv3SnapshotDirectoryVisibleFalse": testAccNetAppVolume_nfsv3SnapshotDirectoryVisibleFalse,
+			"requiresImport":                     testAccNetAppVolume_requiresImport,
+			"complete":                           testAccNetAppVolume_complete,
+			"update":                             testAccNetAppVolume_update,
+			"updateSubnet":                       testAccNetAppVolume_updateSubnet,
+			"updateExportPolicyRule":             testAccNetAppVolume_updateExportPolicyRule,
+		},
+		"DataSource": {
+			"basic": testAccDataSourceNetAppVolume_basic,
+		},
+	}
```

Comment:
why are we doing this? they do not conflict with each other

Reply:
@katbyte , because all of failed TCs are caused by resource quota issue. To resolve this issue, there are two solutions. One solution is to run these TCs in sequence. So I made this PR. Another one solution is to increase the quota. If you prefer the second solution, could you increase the quota and I will close this PR? Make sense?

The error message in Teamcity:
```
Failure sending request: StatusCode=0 -- Original Error: autorest/azure: Service returned an error. Status=<nil> Code="ResourceQuotaExceeded" Message="Creating the resource of type netAppAccounts would exceed the quota of 10 resources of type netAppAccounts per subscription. The current resource count is 10, please delete some resources of this type before creating a new one."
```

Reply:
Sure. Updated PR.